### ISSUE TITLE
Didn't find class "com.bun.miitmdid.core.JLibrary"

### ISSUE THREAD

**Issue by terezo** [AppsFlyerSDK/appsflyer-android-sdk#13]
Hi there,

I am trying to integrate your SDK and keeps getting following error, after calling  `AppsFlyerLib.getInstance().startTracking(this);` in my `App.class`

Dependencies:
`	implementation 'com.appsflyer:af-android-sdk:5.4.1'
	implementation 'com.android.installreferrer:installreferrer:2.1'
`

`2020-07-14 13:24:20.623 I/AppsFlyerOaid5.4.0: Fetch
    java.lang.NoClassDefFoundError: Failed resolution of: Lcom/bun/miitmdid/core/JLibrary;
        at com.appsflyer.oaid.OaidClient.fetchMsa(OaidClient.java:76)
        at com.appsflyer.oaid.OaidClient.fetch(OaidClient.java:64)
        at com.appsflyer.AppsFlyerLibCore.ɩ(:50602)
        at com.appsflyer.AppsFlyerLibCore.Ι(:1603)
        at com.appsflyer.AppsFlyerLibCore.ι(:99)
        at com.appsflyer.AppsFlyerLibCore$e.run(:3307)
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:458)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:301)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
        at com.appsflyer.AFExecutor$1$1.run(:31)
        at java.lang.Thread.run(Thread.java:764)
     Caused by: java.lang.ClassNotFoundException: Didn't find class "com.bun.miitmdid.core.JLibrary" on path: DexPathList[[zip file "/system/framework/org.apache.http.legacy.boot.jar", zip file "/data/app/sk.tb.ib.tatraandroid.dev-QICVxyapa-pn_lsrS9P07A==/base.apk"],nativeLibraryDirectories=[/data/app/sk.tb.ib.tatraandroid.dev-QICVxyapa-pn_lsrS9P07A==/lib/arm64, /data/app/sk.tb.ib.tatraandroid.dev-QICVxyapa-pn_lsrS9P07A==/base.apk!/lib/arm64-v8a, /system/lib64, /vendor/lib64]]
        at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:134)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:379)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
        at com.appsflyer.oaid.OaidClient.fetchMsa(OaidClient.java:76) 
        at com.appsflyer.oaid.OaidClient.fetch(OaidClient.java:64) 
        at com.appsflyer.AppsFlyerLibCore.ɩ(:50602) 
        at com.appsflyer.AppsFlyerLibCore.Ι(:1603) 
        at com.appsflyer.AppsFlyerLibCore.ι(:99) 
        at com.appsflyer.AppsFlyerLibCore$e.run(:3307) 
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:458) 
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) 
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:301) 
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167) 
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641) 
        at com.appsflyer.AFExecutor$1$1.run(:31) 
        at java.lang.Thread.run(Thread.java:764) 
`

Do you have any idea what migh be wrong? Thanks

**Comment by afdima** [comment]
False-positive. Will be fixed in v5.4.2.

**Comment by pedpess** [comment]
@afdima hi :), instead of opening a new issue, I would like to just to mention that there's another falsy warning showing in the newest 6.20.1 version as well. I added in my Android proguard file to ignore this one for now.

```
> Task :app:transformClassesAndResourcesWithR8ForRelease
AGPBI: {"kind":"warning","text":"Missing class: com.bun.miitmdid.interfaces.IIdentifierListener","sources":[{}],"tool":"R8"}
```
